### ISSUE TITLE
Creating a Retro Timer

### ISSUE THREAD

**Issue by theokoles7** [FordLabs/retroquest#349]
#### Context
We use a custom fork of RetroQuest at the company that I work for. Somebody within the company suggested creating a configurable timer to be placed on the retro board. I've created the front end code for one, however, I'm still pretty new to working in Angular, and I'm confused on how to create the back end functionality that would allow the same timer to be visible for every client/user viewing the board. 

#### Story
**As a** developer
**I need** some help completing my design for a retro timer
**So that** users can have a visual representation of how much time they have left to add thoughts/comments before the retrospective session begins

#### Acceptance Criteria
**When** the timer is started
**Then** it should visibly count down on every client/user's view

#### Design
Here is the link (https://github.com/theokoles7/retroquest-timer.git) to the repo that I made for tracking my progress on this project. The repo contains only the component(s) that I created. Not the entire RQ app.

**Comment by theokoles7** [comment]
I'm assuming that the best way to implement this is going to be adding a timer attribute of some sort to the Board.java class file, and calling that variable through one of the services? I would test this out instead of asking, but we run the app on a CentOS7 machine with no gui, so I'm trying to avoid wasting a lot of time going back and forth between files in the terminal.

**Comment by LowBudgetMan** [comment]
Super cool you want to contribute back! Quick question, is this related to #12 at all or are you thinking of using the timer for a different function? I'd be more than happy to walk through any design ideas you have for how to get the timer synced and hooked up, though if you have the frontend code kinda working already, any thoughts on opening a pull request with just the frontend timer code? We have the large view for retro items that we tend to use when discussing those items that we could put this in. It wouldn't be synced across devices, but if the host is sharing their screen it could function similarly.

**Comment by theokoles7** [comment]
Hi! In reference to #12 , I'll admit that I hadn't seen that before, but this enhancement that I've mentioned is slightly different. It would be the same location/functionality, but for our use case, we want it to serve as a countdown for how much time users have left to add comments/thoughts/action items **before** they start going over them. So, for example, if my team starts their retro meeting at 12:00, and agree that they'll start going over action items and thoughts at 12:10, then they can set the timer for 10 minutes to give everyone something to track how much time they have left to add content. The acceptance criteria of what I'm working on doesn't require those features to be disabled when the timer goes off or anything like that. 

As far as making a pull request, I'm not sure how that would work. What I have in my repo was a "scratch Angular project" that I made specifically to create and design the two timer components that I made. It's not the whole RQ app. However, I'm perfectly fine with someone using the front end code that I've written. I have no problem with that. 

![image](https://user-images.githubusercontent.com/61356329/140560509-e6bbcb75-ecef-461d-8a82-010e202834e8.png)
